repos/system_upgrade/common/actors/udev/udevadminfo/libraries/udevadminfo.py

```diff
@@ -0,0 +1,19 @@
+from leapp.exceptions import StopActorExecutionError
+from leapp.libraries.stdlib import api, CalledProcessError, run
+from leapp.models import UdevAdmInfoData
+
+
+def process():
+    try:
+        out = run(['udevadm', 'info', '-e'])['stdout']
+    except CalledProcessError as err:
```

Comment:
@tomasfratrik  btw, totally correct would be
```suggestion
    except CalledProcessError, OSError as err:
```
as in case the command cannot be executed / executable does not exist, `OSError` is raised instead. But in this case, we are user that `udevadm` will be always present and executable on the system, so `CalledProcessError` is enough in this case. Just saying, so you are aware of the difference.

Reply:
Done.